pkg/database/document_database.go

```diff
@@ -303,7 +303,7 @@ func (d *db) SearchDocuments(ctx context.Context, req *protomodel.DocumentSearch
 		return nil, fmt.Errorf("invalid offset")
 	}
 
-	reader, err := d.documentEngine.GetDocuments(ctx, req.Collection, req.Query, int64(offset))
+	reader, err := d.documentEngine.GetDocuments(ctx, req.Collection, req.Query, int64(offset), req.OrderBy)
```

Comment:
minor detail, but I'd put orderBy before the offset so to have the same ordering as in sql